### PR TITLE
Procedure Explained

### PR DESCRIPTION
**Pull request by V1NAY8**

### CODE REVIEW

**Comment by rakhi2104** [comment]
Thanks for your contribution @V1NAY8

**Comment by V1NAY8** [comment]
Anything more needed?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
-# GCP_RaspberryPi
-Python Code to Push images captured by Raspberry Pi to Google Cloud Storage
+# Connection between Google Cloud Storage and Raspberry Pi 3B
+
+## Target
+- Raspberry pi is most commonly used device in IoT Platform
```
Comment (rakhi2104) [inline README.md]: 'Raspberry Pi' has got some branding guideline - the P in Pi should be capitalized.
read more [here](https://www.raspberrypi.org/trademark-rules/)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,29 @@
-# GCP_RaspberryPi
-Python Code to Push images captured by Raspberry Pi to Google Cloud Storage
+# Connection between Google Cloud Storage and Raspberry Pi 3B
+
+## Target
+- Raspberry pi is most commonly used device in IoT Platform 
+- To do efficient Analytics on Data that is captured from Raspberry pi we transfer the data to high capability Server.
```
Comment (rakhi2104) [inline README.md]: don't capitalize words unnecessarily, follow English rules

File: README.md [hunk 3]
```diff
@@ -1,2 +1,29 @@
...
+## Packages Required
+- Python 3.7 (Preferably above by the time you read this!)
+- Google cloud Storage.
+- PIL (For Image Transformations)
+- picamera ( For capturing Images in Raspberry pi)
+
+
+## Procedure
+### Credentials Required for Authentication.
+- Create a Google account and proceed to [Google-Storage](https://console.cloud.google.com)
```
Comment (rakhi2104) [inline README.md]: sanitize English

**Comment by rakhi2104** [comment]
> Anything more needed?

Please make changes according to the reviews

**Review by rakhi2104 — COMMENTED** [review]
Update content according to reviews

**Pull request closed without merging**